### PR TITLE
Wise config support changes

### PR DESCRIPTION
**Pull request by BryceV**
#548 work
- Support for parsing json files 
- You can now store the config in JSON form in redis
- Started work for async pubsub when config it is stored in redis.  
- Created a simple README to log how to pass in a redis url for getting the config

### CODE REVIEW

File: wiseService/wiseService.js [hunk 1]
```diff
@@ -847,12 +846,12 @@ app.get('/stats', [noCacheJson], function (req, res) {
   res.send(stats);
 });
 // ----------------------------------------------------------------------------
-if (getConfig('wiseService', 'regressionTests')) {
-  app.post('/shutdown', (req, res) => {
+app.post('/shutdown', (req, res) => {
```
Comment (awick) [inline wiseService/wiseService.js:849]: Why the change? I think not having api available is better.

File: tests/config.test.json [hunk 2]
```diff
@@ -0,0 +1,242 @@
+{
```
Comment (awick) [inline tests/config.test.json:1]: I would say until we have editing available we should should use the ini one, unless this is just for a test.

File: wiseService/package.json [hunk 3]
```diff
@@ -17,7 +17,7 @@
     "build": "npm run bundle:min",
     "start": "node wiseService.js -c ../tests/config.test.ini",
     "prestart": "npm run bundle:min",
-    "dev": "nodemon wiseService.js -c ../tests/config.test.ini & webpack -d --watch --progress --hide-modules --display-error-details --config vueapp/build/webpack.dev.conf.js",
+    "dev": "nodemon wiseService.js -c ../tests/config.test.json & webpack -d --watch --progress --hide-modules --display-error-details --config vueapp/build/webpack.dev.conf.js",
```
Comment (awick) [inline wiseService/package.json:20]: until editing is available lets use the ini one

File: tests/config.test.json (hunk 2 above)
Comment (BryceV) [reply tests/config.test.json:1]: I left the json version for testing

File: wiseService/wiseService.js (hunk 1 above)
Comment (BryceV) [reply wiseService/wiseService.js:849]: 👍

File: wiseService/package.json (hunk 3 above)
Comment (BryceV) [reply wiseService/package.json:20]: Ok
